src/widget/wtrackproperty.h

```diff
@@ -10,15 +10,21 @@
 #include "track/track.h"
 #include "widget/trackdroptarget.h"
 #include "widget/wlabel.h"
+#include "widget/wtrackmenu.h"
 
 class WTrackProperty : public WLabel, public TrackDropTarget {
     Q_OBJECT
   public:
-    WTrackProperty(const char* group, UserSettingsPointer pConfig, QWidget* pParent);
+    WTrackProperty(const char* group,
+            UserSettingsPointer pConfig,
+            QWidget* pParent,
```

Comment:
Placement of `parent` parameter

Reply:
To clarify, by convention we put `parent` as the first parameter for constructors.